lyonza/WhiteTextBroadcast.py

```diff
@@ -0,0 +1,329 @@
+from pathlib import Path
+from re import findall
+
+from modules.CardType import CardType
+from modules.Debug import log
+from modules.RemoteFile import RemoteFile
+
+class WhiteTextBroadcast(CardType):
+    """
+    This class describes lyonza's CardType based on Wvdh's "WhiteTextBroadcast" card to show SxxExx format instead of absolute numbering
+    """

```

Comment:
```suggestion
    This class describes lyonza's CardType based on Wvdh's
    "WhiteTextBroadcast" card to show SxxExx format instead of absolute numbering.
    """

```